docs/howto_doc.md

```diff
@@ -0,0 +1,17 @@
+# How to generate documentation locally.
```

Comment:
Do you think a `README.md` file inside the `docs/` folder would be a good idea?

We can transfer this content to the readme.

Reply:
Sure. We can do that. But if we do that there will be 2 files with the same data. If we have the README.md file, I think we can omit the howto_doc.md file. That works out ?